daemon/graphdriver/devmapper/driver.go

```diff
@@ -151,7 +151,7 @@ func (d *Driver) Remove(id string) error {
 
 	// This assumes the device has been properly Get/Put:ed and thus is unmounted
 	if err := d.DeviceSet.DeleteDevice(id, false); err != nil {
-		return err
+		return fmt.Errorf("failed to remove device %v:%v", id, err)
```

Comment:
@thaJeztah Tried to have a quick look at errors.Wrapf() but it is not obvious to me what are we gaining by using errors.Wrapf() instead of fmt.Errorf(). Can you elaborate a bit.